albumentations/augmentations/transforms.py

```diff
@@ -1633,8 +1633,12 @@ def get_params(self):
 
         return {"quality": random.randint(self.quality_lower, self.quality_upper), "image_type": image_type}
 
-    def get_transform_init_args_names(self):
-        return ("quality_lower", "quality_upper", "compression_type")
+    def get_transform_init_args(self):
+        return {
+            "quality_lower": self.quality_lower,
+            "quality_upper": self.quality_upper,
+            "compression_type": self.compression_type.value,
```

Comment:
May be a source of error, because `compression_type` would be an `int`. So we need force cast to `int` or to `ImageCompressionType` in the `__init__` method.

Reply:
@Dipet Done